src/components/Reservation.jsx

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import '../styles/carousel-item.scss';
+import formatDate from '../utils/formatDate';
+import formatToCurrency from '../utils/formatToCurrency';
+
+const Reservation = ({ data }) => {
+  const {
+    days, total_price: totalPrice, created_at: createdAt, videogame,
+  } = data;
+  const {
+    id, name, photo, description,
+  } = videogame;
+
+  return (
+    <article className="col">
+      <Link key={id} to={`/details/${id}`}>
+        <div className="vh-25 mb-4">
+          <img className="h-100 img-fluid rounded aspect-ratio-1" src={photo} alt={name} />
+        </div>
+        <h3 className="h6 fw-bolder text-uppercase mb-4">{name}</h3>
+      </Link>
+      <div className="text-muted small">
+        <p className="mb-0">
+          <span className="fw-bold">Starting date:</span>
+          {' '}
+          {formatDate(createdAt)}
+        </p>
+        <p className="mb-0">
+          <span className="fw-bold">Finishing date:</span>
+          {' '}
+          {formatDate(createdAt, days)}
+        </p>
+        <p className="mb-0">
+          <span className="fw-bold">Days:</span>
+          {' '}
+          {days}
+        </p>
+        <p className="mb-0">
+          <span className="fw-bold">Total price:</span>
+          {' '}
+          {formatToCurrency(totalPrice)}
+        </p>
+        <p className="d-webkit-box clamp-3">{description}</p>
```

Comment:
The formatting of reservation details is repeated a few times with slight differences. You could create a small sub-component or a function that formats this data for you to make the code DRY. For example: 

  

```jsx 

const Detail = ({label, value}) => ( 

  <p className="mb-0"> 

    <span className="fw-bold">{label}:</span> 

    {' '} 

    {value} 

  </p> 

); 

``` 

  

And use it in your component: 

  

```jsx 

<Detail label="Starting date" value={formatDate(createdAt)} /> 

<Detail label="Finishing date" value={formatDate(createdAt, days)} /> 

<Detail label="Days" value={days} /> 

<Detail label="Total price" value={formatToCurrency(totalPrice)} /> 

```